cli/tests/pcluster/api/controllers/test_cluster_compute_fleet_controller.py

```diff
@@ -40,27 +39,6 @@ def _build_dynamodb_item(scheduler, status, last_status_updated_time=None):
             else:
                 last_status_updated_time = str(datetime.now())
             dynamodb_item["Item"]["LastUpdatedTime"] = last_status_updated_time
-    elif scheduler == "plugin":
-        compute_fleet_status_manager = JsonComputeFleetStatusManager("cluster-name")
-
-        if status == "UNKNOWN":
-            dynamodb_item = {}  # Test dynamodb item not exist
-        else:
-            dynamodb_item = {
-                "Item": {
-                    compute_fleet_status_manager.DB_DATA: {
-                        compute_fleet_status_manager.COMPUTE_FLEET_STATUS_ATTRIBUTE: status
-                    }
-                }
-            }
-            if last_status_updated_time:
-                last_status_updated_time = str(last_status_updated_time)
-            else:
-                last_status_updated_time = str(datetime.now())
-
-            dynamodb_item["Item"][compute_fleet_status_manager.DB_DATA][
-                compute_fleet_status_manager.COMPUTE_FLEET_LAST_UPDATED_TIME_ATTRIBUTE
-            ] = last_status_updated_time
```

Comment:
`get_manager(` still returns a `JsonComputeFleetStatusManager `
 so either we keep the test and adapt it to run without the SchedulerPluign or we change the `get_manager` and remove the option to get a  `JsonComputeFleetStatusManager ` if it is not used anymore.

Reply:
You're right. Code updated to keep testing that branch